.gitattributes

```diff
@@ -72,7 +72,7 @@ tools/mantis2gh_stripped.csv typo.missing-header
 /.depend.menhir          typo.prune
 
 # Makefiles may contain tabs
-Makefile*                typo.tab=may
+Makefile*                typo.makefile=may
```

Comment:
I would prefer `makefile-tab` or `tab-makefile` or `makefile-whitespace` to just `makefile`, in case we want to add other makefile-specific rules in the future, or to highlight that this rule is just about whitespace.

Reply:
`makefile-whitespace` seems nice to me - I've changed it